### PR TITLE
jomungandr: Fix attributs in swagger

### PR DESCRIPTION
**Pull request by kadhikari**
**Tickets:** 
https://jira.kisio.org/browse/NAVITIAII-2415

**Attributs:**

- embedded_type
- gender
- transfer_type
- data_freshness
- mode=StreetNetworkMode
- type=SectionType

### CODE REVIEW

**Comment by kadhikari** [comment]
Is_frequece modified to boolean

File: source/jormungandr/jormungandr/interfaces/v1/serializer/base.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def to_value(self, value):
 
     @staticmethod
     def _get_all_possible_values(pb_type):
-        return [v.name for v in pb_type.DESCRIPTOR.values]
+        return [v.name.lower() for v in pb_type.DESCRIPTOR.values]
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/base.py:134]: I think that we keep a few enum in upper case.

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 2]
```diff
@@ -702,7 +702,7 @@ def output(self, key, value):
 
 route = deepcopy(generic_type)
 route["links"] = DisruptionLinks()
-route["is_frequence"] = fields.String
+route["is_frequence"] = fields.Boolean()
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py]: This look like a non backward compatible change

**Review by kinnou02 — CHANGES_REQUESTED** [review]

**Review by mbounabi — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/base.py (hunk 1 above)
Comment (kadhikari) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/base.py:134]: As we discussed yesterday, among all the enum  types, we have some in uppercase where all the attributs are lowercase (forced in the code). What should i do? manage some in uppercase and some in lowercase, all in lowercase or all in uppercase?

File: source/jormungandr/jormungandr/interfaces/v1/fields.py (hunk 2 above)
Comment (kadhikari) [reply source/jormungandr/jormungandr/interfaces/v1/fields.py]: Test swagger fails without this change.

Comment (kinnou02) [reply source/jormungandr/jormungandr/interfaces/v1/fields.py]: In production we return a string, not a bool:
```
"is_frequence": "False",
```
it's ugly but we can't change it without breaking thing.

File: source/jormungandr/jormungandr/interfaces/v1/serializer/base.py (hunk 1 above)
Comment (kinnou02) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/base.py:134]: https://github.com/CanalTP/navitia/pull/2408/files#diff-29d0a12a94cc6560194088f98fcd72f3R130
so it's ok to lower every values

**Review by nonifier — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

**Pull request merged**